python/lsst/ts/Dome/mock_controller.py

```diff
@@ -338,24 +341,20 @@ async def crawlEl(self, **kwargs):
 
         # No conversion from radians to degrees needed since both the commands and the mock az controller
         # use radians.
-        await self.lwscs.crawlEl(velocity=float(kwargs["elRate"]))
+        await self.lwscs.crawlEl(velocity=float(kwargs["velocity"]))
 
-    async def setLouver(self, **kwargs):
-        """Set the position of a louver.
+    async def setLouvers(self, **kwargs):
```

Comment:
Picky point but... you have `move_az` but `setLouvers` in this class and `moveAz` in another class. If there isn't some underlying logic that drives these choices please consider standardizing. It would make your already clear code even easier to follow.

Reply:
Thanks for being picky!

Reply:
Done